libs/metrics/top_level_metrics.py

```diff
@@ -204,17 +196,17 @@ def calculate_case_density(
 
 
 def calculate_test_positivity(
-    positive_tests: pd.Series, negative_tests: pd.Series, smooth: int = 7, lag_lookback: int = 7
+    region_dataset: OneRegionTimeseriesDataset, lag_lookback: int = 7
 ) -> pd.Series:
-    """Calculates positive test rate.
-
-    Args:
-        positive_tests: Number of cumulative positive tests.
-        negative_tests: Number of cumulative negative tests.
+    """Calculates positive test rate from combined data."""
```

Comment:
Add arguments? (i realize we didn't have lag_lookback in the last docstring either)